README.md

```diff
@@ -104,7 +104,9 @@ save you having to request it through the ServiceFactory every time.
 
 #### Timezone support
 By default, Doctrine2 assumes all DateTimes are in UTC.  
-You can force all dates going in to the DB to be converted to UTC and all DateTimes coming out to have UTC set by adding the following configuration to your YAML files.
+You can force all dates going in to the DB to be converted to UTC and all DateTimes coming out to have UTC set by adding the following configuration to your YAML files.  
+If using this configuration, DateTimes passed in to the service must be a [Chronos](https://book.cakephp.org/3.0/en/chronos.html) object.  
```

Comment:
That makes it a pain to check if the interface is mutable or not. We'd need to clone it if it was mutable.